plugins/modules/web_infrastructure/jira.py

```diff
@@ -149,6 +150,32 @@
     default: true
     type: bool
 
+  attachment:
+    type: dict
+    required: false
+    aliases: [file]
+    description:
+      - Information about the attachment being uploaded
+    suboptions:
+      filename:
+        required: true
+        type: str
+        description: >
+          The path to the file to upload (from the remote node) or, if C(content) is specified,
```

Comment:
```suggestion
          The path to the file to upload (from the remote node) or, if I(content) is specified,
```
(See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation)